hygieia/.openshift/templates/docker-build-git.yml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
```

Comment:
is this the same as the new `build-docker-generic` template?

Reply:
probably, but since we currently don't have that in a release, I'd rather no point to it yet.